opentelemetry-proto/src/transform/common.rs

```diff
@@ -42,21 +42,47 @@ pub mod tonic {
     #[cfg(any(feature = "trace", feature = "logs"))]
     use opentelemetry_sdk::Resource;
 
-    impl From<opentelemetry_sdk::InstrumentationLibrary> for InstrumentationScope {
-        fn from(library: opentelemetry_sdk::InstrumentationLibrary) -> Self {
+    impl
+        From<(
+            opentelemetry_sdk::InstrumentationLibrary,
+            Option<Cow<'static, str>>,
+        )> for InstrumentationScope
+    {
+        fn from(
+            data: (
+                opentelemetry_sdk::InstrumentationLibrary,
+                Option<Cow<'static, str>>,
+            ),
+        ) -> Self {
+            let (library, target) = data;
             InstrumentationScope {
-                name: library.name.into_owned(),
+                name: target
+                    .map(|t| t.replace("::", "."))
```

Comment:
please do this in appender only.

Reply:
Not doing it on purpose. Not all exporters would like to use the target as scope, and mayn't want this conversion.

Reply:
ok, the conversion has been removed from the exporter now.